en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-inbound-endpoint-example.md

```diff
@@ -0,0 +1,110 @@
+       <parameters>
+           <parameter name="inbound.behavior">listening</parameter>
+           <parameter name="sequential">true</parameter>
+           <parameter name="coordination">true</parameter>
+           <parameter name="port">5000</parameter>
+           <parameter name="headerLength">2</parameter>
+           <parameter name="isProxy">false</parameter>
+       </parameters>
+   </inboundEndpoint>
+   ```
+   Sequence to process the message.
+   
+   In this example for simplicity we will just log the message, but in a real world use case, this can be any type of message mediation.
+ 
+   ```xml
+   <?xml version="1.0" encoding="UTF-8"?><sequence xmlns="http://ws.apache.org/ns/synapse" name="requestISO" onError="fault">
+       <log level="full">
+           <property name="Log_Message for ISO8583 Inbound Endpoint" value="Message received from sample1-source"/>
+       </log>
+   </sequence>
+   ```
+## Exporting Integration Logic as a CApp
+
+**CApp (Carbon Application)** is the deployable artefact on the Enterprise Integrator runtime. Let us see how we can export integration logic we developed into a CApp. To export the `Solution Project` as a CApp, a `Composite Application Project` needs to be created. Usually, when a solution project is created, this project is automatically created by Integration Studio. If not, you can specifically create it by navigating to  **File** -> **New** -> **Other** -> **WSO2** -> **Distribution** -> **Composite Application Project**. 
+
+1. Right click on Composite Application Project and click on **Export Composite Application Project**.</br> 
+  <img src="/assets/img/connectors/capp-project1.png" title="Export as a Carbon Application" width="300" alt="Export as a Carbon Application" />
+
+2. Select an **Export Destination** where you want to save the .car file. 
+
+3. In the next **Create a deployable CAR file** screen, select inbound endpoint and sequence artifacts and click **Finish**. The CApp will get created at the specified location provided in the previous step. 
+
+## Deployment
+
+1. Navigate to the [connector store](https://store.wso2.com/store/assets/esbconnector/list) and search for `ISO8583`. Click on `ISO8583 Inbound Endpoint` and download the .jar file by clicking on `Download Inbound Endpoint`. Copy this .jar file into  <PRODUCT-HOME>/lib folder. 
+
+2. Download [jpos-1.9.4.jar]((http://mvnrepository.com/artifact/org.jpos/jpos/1.9.4)), [jdom-1.1.3.jar]((http://mvnrepository.com/artifact/org.jdom/jdom/1.1.3), and [commons-cli-1.3.1.jar]((http://mvnrepository.com/artifact/commons-cli/commons-cli/1.3.1) and add it to <PRODUCT-HOME>/lib folder.
+
+3. Copy the exported carbon application to the <PRODUCT-HOME>/repository/deployment/server/carbonapps folder. 
+
+4. [Start the WSO2 EI server](https://docs.wso2.com/display/EI660/Running+the+Product). 
```

Comment:
Shall we point to 7.0.0 docs here?

Reply:
Pointed to https://ei.docs.wso2.com/en/latest/micro-integrator/overview/quick-start-guide/#start-the-micro-integrator.